src/commands/prepare.ts

```diff
@@ -95,12 +100,20 @@ export const builder: CommandBuilder = (yargs: Argv) =>
       description: 'The git remote to use when pushing',
       type: 'string',
     })
+    .option('config-from', {
+      description: 'Load .craft.yml from the specified remote branch instead of local file',
+      type: 'string',
+    })
+    .option('calver-offset', {
```

Comment:
Why do we need this?

Reply:
For us, never as we always use the default value `14`. But since Craft is meant to be a generic tool, some other folks may want to tweak this value.